### ISSUE TITLE
0.0.4 can't find src\ios\Purchases.framework'

### ISSUE THREAD

**Issue by solojuve1897** [RevenueCat/cordova-plugin-purchases#4]
Problem with 0.0.4.

On Mac:
0.0.3 - installs and works without issues.
0.0.4 - installs but loading the framework doesn't work (see console-output).

On Windows:
0.0.3 - doesn't install. I believe the purpose of 0.0.4 was to fix that problem.
0.0.4 - installs but gives an error in the console.

From the console:

```
ENOENT: no such file or directory, stat '..\app\node_modules\cordova-plugin-purchases\src\ios\Purchases.framework'
[error] Error running update: { Error: ENOENT: no such file or directory, stat ..\app\node_modules\cordova-plugin-purchases\src\ios\Purchases.framework'
    at Object.statSync (fs.js:851:3)
    at Object.statSync (..\app\node_modules\graceful-fs\polyfills.js:295:24)
    at Object.copySync (..\app\node_modules\@capacitor\cli\node_modules\fs-extra\lib\copy-sync\copy-sync.js:31:86)
    at frameworks.map (..\app\node_modules\@capacitor\cli\dist\ios\update.js:274:22)
    at Array.map (<anonymous>)
    at cordovaPlugins.map.p (..\app\node_modules\@capacitor\cli\dist\ios\update.js:272:20)
    at Array.map (<anonymous>)
    at copyPluginsNativeFiles (..\app\node_modules\@capacitor\cli\dist\ios\update.js:240:20)
    at Object.updateIOS (..\app\node_modules\@capacitor\cli\dist\ios\update.js:47:9)
  errno: -4058,
  syscall: 'stat',
  code: 'ENOENT',
  path:
   '..\app\\node_modules\\cordova-plugin-purchases\\src\\ios\\Purchases.framework' }
```

**Comment by vegaro** [comment]
Looks like the download script assumes for a location of node_modules and plugins folder that might not be the same for everyone. I removed the hook to trigger the script and will be running the script before releasing on our end. I fixed it in master (haven't released it yet)

**Comment by solojuve1897** [comment]
Seems to be fixed by 0.0.6
